pkg/options/rawyamltmpl/raw_text_maker.go

```diff
@@ -28,25 +28,23 @@ import (
 	bundle "github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/apis/bundle/v1alpha1"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/converter"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/filter"
-	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/maker"
+	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/options"
 )
 
-// Maker makes components via go-templating, for objects stored as raw strings
+// Applier applies options via go-templating, for objects stored as raw strings
```

Comment:
What ever happened to using Kinflate or whatever it's called now?

Reply:
Kustomize (formerly Kinflate) could be another options applier, for sure. I haven't added it yet and might not for a little while. Just using go templates should get us some mileage before trying to add new customization systems.